src/py/flwr/client/supernode/app.py

```diff
@@ -66,14 +66,7 @@ def run_supernode() -> None:
 
     # Exit if unsupported argument is passed by the user
     if args.app is not None:
-        log(
-            ERROR,
-            "The `app` argument is deprecated. The SuperNode now automatically "
-            "uses the ClientApp delivered from the SuperLink. Providing the app "
-            "directory manually is no longer supported. Please remove the `app` "
-            "argument from your command.",
-        )
-        sys.exit(1)
+        flwr_exit(ExitCode.SUPERNODE_REMOVED_APP_ARGUMENT)
```

Comment:
Instead of using `flwr_exit` here, should we just make a breaking change and remove `--app` entirely? (i.e., not more warnings)